<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VectorGraphicsRenderingSystemClass</class>
 <widget class="QMainWindow" name="VectorGraphicsRenderingSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VectorGraphicsRenderingSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>580</width>
         <height>293</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>     File     </string>
    </property>
    <addaction name="createFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
   </widget>
   <widget class="QMenu" name="menu_Draw">
    <property name="title">
     <string>     Draw     </string>
    </property>
    <addaction name="line"/>
    <addaction name="polyline"/>
    <addaction name="ellipse"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>     Edit     </string>
    </property>
    <addaction name="choose"/>
    <addaction name="move"/>
    <addaction name="changeColor"/>
   </widget>
   <widget class="QMenu" name="menu_Tips">
    <property name="title">
     <string>     Tips     </string>
    </property>
    <addaction name="Tips"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Draw"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tips"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createFile"/>
   <addaction name="openFile"/>
   <addaction name="saveFile"/>
   <addaction name="separator"/>
   <addaction name="line"/>
   <addaction name="polyline"/>
   <addaction name="ellipse"/>
   <addaction name="separator"/>
   <addaction name="choose"/>
   <addaction name="move"/>
   <addaction name="changeColor"/>
   <addaction name="separator"/>
   <addaction name="Tips"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="createFile">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/new.png</normaloff>:/VectorGraphicsRenderingSystem/res/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="openFile">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/open.png</normaloff>:/VectorGraphicsRenderingSystem/res/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/save.png</normaloff>:/VectorGraphicsRenderingSystem/res/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="line">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/line.png</normaloff>:/VectorGraphicsRenderingSystem/res/line.png</iconset>
   </property>
   <property name="text">
    <string>线段</string>
   </property>
  </action>
  <action name="polyline">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/polyline.png</normaloff>:/VectorGraphicsRenderingSystem/res/polyline.png</iconset>
   </property>
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="ellipse">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/ellipse.png</normaloff>:/VectorGraphicsRenderingSystem/res/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="choose">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/choose.png</normaloff>:/VectorGraphicsRenderingSystem/res/choose.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="move">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/move.png</normaloff>:/VectorGraphicsRenderingSystem/res/move.png</iconset>
   </property>
   <property name="text">
    <string>平移</string>
   </property>
  </action>
  <action name="changeColor">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/color.png</normaloff>:/VectorGraphicsRenderingSystem/res/color.png</iconset>
   </property>
   <property name="text">
    <string>改色</string>
   </property>
  </action>
  <action name="Tips">
   <property name="icon">
    <iconset resource="VectorGraphicsRenderingSystem.qrc">
     <normaloff>:/VectorGraphicsRenderingSystem/res/tips.png</normaloff>:/VectorGraphicsRenderingSystem/res/tips.png</iconset>
   </property>
   <property name="text">
    <string>用户提示</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VectorGraphicsRenderingSystem.qrc"/>
 </resources>
 <connections/>
</ui>
